WIP-CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Add networkpolicy generator chart with common rules
 - Added option to enable harbor GC via config
 - Added openstack api metrics.
+- Added Swift type for rclone-sync and bucket type on buckets for rclone-sync.
```

Comment:
Maybe this could be made a bit clearer, like
```suggestion
- Added support for Swift in rclone-sync.
  - It is now possible to specify type (S3/Swift) per bucket.
```
?

Reply:
Yeah that is better. I'll change it to that before merging.